README.md

```diff
@@ -61,21 +77,20 @@ in the [mdn/content repository](https://github.com/mdn/content).
 
 ## How it works
 
-Yari is multiple things but at its core is the MDN content as `index.html`
-files, in `git`, that contain the metadata (as front-matter) and
-the bulk of the document.
+repo as `index.html` files that contain metadata as YAML front-matter followed
```

Comment:
This sentence looks like something went wrong.

Reply:
Yup, it did!

It was supposed to read as follows:

Yari does a number of things, the most important of which is to render and serve the MDN content found in the [content repo](https://github.com/mdn/content). Each document is stored as an `index.html` file that contains metadata presented as YAML [front-matter](https://github.com/mdn/content#fundamental-concepts) followed by the document source.

I'll update this in the next push.